src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/NewAzureCognitiveServicesAccountApiProperty.cs

```diff
@@ -0,0 +1,34 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Management.CognitiveServices.Models;
+using System.Management.Automation;
+using CognitiveServicesModels = Microsoft.Azure.Management.CognitiveServices.Models;
+
+namespace Microsoft.Azure.Commands.Management.CognitiveServices
+{
+    /// <summary>
+    /// Generate Cognitive Services Account ApiProperties class
+    /// </summary>
+    [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "CognitiveServicesAccountApiProperty", SupportsShouldProcess = true), OutputType(typeof(CognitiveServicesModels.CognitiveServicesAccountApiProperties))]
+    public class NewAzureCognitiveServicesAccountApiPropertiesCommand : CognitiveServicesAccountBaseCmdlet
+    {
+        public override void ExecuteCmdlet()
+        {
+            base.ExecuteCmdlet();
+
+            WriteObject(new CognitiveServicesAccountApiProperties());
```

Comment:
if the cmdlet just create a local object, no need to support should process here.